### ISSUE TITLE
Modules and Plugins with Known Issues Doc Update: Add Divi theme

### ISSUE THREAD

**Issue by sparklingrobots** [pantheon-systems/documentation#5348]
Re: [Modules and Plugins with Known Issues](https://pantheon.io/docs/modules-plugins-known-issues)

Priority: Low

## Issue Description:
This entire post is from Rob Watson on our Community Slack: 
If anyone else who uses the Divi theme by Elegant Themes is having trouble using the Visual Builder after about version 4.0.5 or 4.0.6 of Divi, I'd love to hear from you. I've been going back and forth for days now with them on chat trying to explain how Pantheon Live hosting is read-only. They simply cannot grasp the concept of having actual security on a WordPress site and they keep telling me to "talk to Pantheon support and ask them to change file permissions" so that Divi, which didn't used to need to write to the filesystem, can now write to it. By way of example, this was the latest "fix" they sent me, which I've a hunch will not work on Pantheon.

> Please follow this tutorial to set proper files and folder permissions:
> https://www.wpbeginner.com/beginners-guide/how-to-fix-file-and-folder-permissions-error-in-wordpress/
> And add this line of code in wp-config.php file:
> define('FS_METHOD', 'direct');
> Please add this line above this line of code:
> /* That's all, stop editing! Happy publishing. */
> I would have added that but I am not allowed to make this change due to permission issue on files and folders. This file exist in the root directory where WordPress installed and you can make this change through ftp or file manager from web hosting panel like cPanel.
> If you are not sure about making these changes then contact your web host to assist you with the changes as they have full control of all the files and folders. They can help you adjust the permission level on your files and folders. Permission level like 775 or 770 or 755 on them should make it work.
> And then you can update theme after that and let me know how it goes. 

Another user suggested this fix: https://wordpress.org/support/topic/ftp_nlist-warnings/

## Suggested Resolution
I have not tried to reproduce the issue. If we're able to reproduce, it would be helpful to have the issue (and the fix, if it works) documented on this page.

**Comment by crobertwatson** [comment]
Solved. I had thought the suggested solution above wasn't working but it turned out that step 3 below had failed to commit for some reason. Once that code committed, and the other steps were in place, the Divi pages could again be edited in the Visual Builder.

1) Add /files/et-cache/ to filesystem.
2) Create et-content symlink pointing to /files/et-cache in /code/wp-content/ per instructions from Pantheon Docs.
3) Add this to wp-config.php.
```
/* For themes and plugins that do not check if the wp_filesystem link is active before trying to use it, which causes the errors.*/
if ( ! defined( 'FS_METHOD' ) ) define( 'FS_METHOD', 'direct' );
```
4) Update to Divi 4.0.6.
5) Test and deploy.

**Comment by EdwardAngert** [comment]
closed by #5362

**Comment by alexfornuto** [comment]
Thanks for your input @crobertwatson!
